lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -106,11 +45,6 @@ def sort_string_by_language(str_literals)
           str_literals.uniq!
           str_literals.delete_if { |s| s.nil? || s.length <= 0 }
         end
-
-        def blank_node?(obj)
```

Comment:
This was a bad way to check for blank nodes.  It is being dropped.  The replacement is obj.anonymous? which uses the ruby-rdf/rdf gem's test for blank nodes.